lua/lscs/autorun/lscs_player.lua

```diff
@@ -87,15 +87,14 @@ if SERVER then
 			end
 
 			return
+		else
```

Comment:
?? nothing broken here ?? allow == true => restore old variables,  allow ~= true => set to 0, are you spamming this in think with nil or something?